src/averageAges.js

```diff
@@ -13,13 +13,30 @@
  * @param {number} century - optional
  *
  * @return {number}
+ *
  */
+function averageAge(objectWithPeoples) {
+  const allAges = objectWithPeoples.map(human => human.died - human.born);
+
+  return allAges.reduce(
+    (sumOFAge, nextAge) => sumOFAge + nextAge,
+    0) / allAges.length;
+}
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const diedInThisCentury = man => century
```

Comment:
4 пункт - https://mateacademy.slack.com/archives/C021CDS2J79/p1624364911131100
```suggestion
  const hasDiedInThisCentury = man => century
```